drivers/bmx280/include/bmx280_internals.h

```diff
@@ -113,7 +113,7 @@ extern "C" {
  * @{
  */
 #define CALIB_T_P_BASE                  (BMX280_DIG_T1_LSB_REG)
-#define CALIB_T_P_LEN                   (17U)
+#define CALIB_T_P_LEN                   (32U) /* might be larger than needed */
```

Comment:
If I'm correct this should be `0x17` or `0d22`. But we can change that in a follow-up with some more thorough testing. `32U` should definitely be enough.